client/modules/_hooks/src/reconportal/useGetEventTypes.ts

```diff
@@ -0,0 +1,31 @@
+import { useSuspenseQuery, useQueryClient } from '@tanstack/react-query';
+import apiClient from '../apiClient';
+
+export type EventTypeResponse = {
+  display_name: string;
+  name: string;
+};
+
+async function getEventTypes() {
```

Comment:
getEventTypes -> getReconPortalEventTypes.  Same for relates types and useGetEvent.